lib/ansible/modules/cloud/cloudstack/cs_physical_network.py

```diff
@@ -0,0 +1,464 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2017, Netservers Ltd. <support@netservers.co.uk>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: cs_physical_network
+short_description: Manages physical networks on Apache CloudStack based clouds.
+description:
+    - Create, update and remove networks.
+    - Enabled and disabled Network Service Providers
+    - Enables Internal LoadBalancer and VPC/VirtualRouter elements as required
+version_added: "2.8"
+author:
+ - Netservers Ltd. (@netservers)
+ - Patryk Cichy (@PatTheSilent)
+options:
+  physical_network:
+    description:
+      - Name of the physical network.
+    required: true
+    aliases:
+      - name
+    type: str
+  zone:
+    description:
+      - Name of the zone in which the network belongs.
+      - If not set, default zone is used.
+    type: str
+  broadcast_domain_range:
+    description:
+      - broadcast domain range for the physical network[Pod or Zone].
+    required: false
+    choices: [ POD, ZONE ]
+    type: str
+  domain:
+    description:
+      - Domain the network is owned by.
+    type: str
+  isolation_method:
+    description:
+      - Isolation method for the physical network.
+    required: false
+    choices: [ VLAN, GRE, L3 ]
+    type: str
+  network_speed:
+    description:
+      - The speed for the physical network.
+    required: false
```

Comment:
Not needed. `required: false` should be removed for all params.

Reply:
Done.